src/main/java/mekanism/common/config/ClientConfig.java

```diff
@@ -0,0 +1,54 @@
+package mekanism.common.config;
+
+import io.netty.buffer.ByteBuf;
+import mekanism.common.config.options.BooleanOption;
+import mekanism.common.config.options.DoubleOption;
+import mekanism.common.config.options.IntOption;
+
+/**
+ * Created by Thiakil on 15/03/2019.
+ */
+public class ClientConfig extends BaseConfig {
+
+    public final BooleanOption enablePlayerSounds = new BooleanOption(this, "client", "EnablePlayerSounds", true,
+          "Play sounds for Jetpack/Gas Mask/Flamethrower (all players).");
+
+    public final BooleanOption enableMachineSounds = new BooleanOption(this, "client", "EnableMachineSounds", true,
+          "If enabled machines play their sounds while running.");
+
+    public final BooleanOption holidays = new BooleanOption(this, "client", "Holidays", true,
+          "Christmas/New Years greetings in chat.");
+
+    public final DoubleOption baseSoundVolume = new DoubleOption(this, "client", "SoundVolume", 1D,
+          "Adjust Mekanism sounds' base volume. < 1 is softer, higher is louder.");
+
+    public final BooleanOption machineEffects = new BooleanOption(this, "client", "MachineEffects", true,
+          "Show particles when machines active.");
+
+    public final BooleanOption enableAmbientLighting = new BooleanOption(this, "client", "EnableAmbientLighting", true,
+          "Should active machines produce block light. Causes chunk redraws!");
```

Comment:
Is this comment still accurate after the lighting changes you guys did?

Reply:
You are right it doesn't still cause chunk redraws, I will fix it in the next commit I make.

Reply:
hmm just looked into it, `mekanism.common.block.BlockMachine#getLightValue` does still change block light, which causes redraws - its just less often

Reply:
It is a lot less often, for Mekanica I didn't have the comment describing that it causes chunk redraws. As with the rate it does it at, it shouldn't really be something for people to worry about.